tests/ui/diagnostic-width/E0271.ascii.stderr

```diff
@@ -7,14 +7,16 @@ LL | |             Err::<(), _>(
 LL | |                 Ok::<_, ()>(
 ...  |
 LL | |     )
-   | |_____^ type mismatch resolving `<Result<Result<(), Result<Result<(), ...>, ...>>, ...> as Future>::Error == Foo`
+   | |_____^ type mismatch resolving `<Result<..., ()> as Future>::Error == Foo`
```

Comment:
Example of the allowlisting of "known short" types so we don't replace them with `...` unnecessarily.